SecurityPkg/SecurityPkg.dec

```diff
@@ -659,6 +659,14 @@
   ## This PCD records LASA field in CC EVENTLOG ACPI table.
   gEfiSecurityPkgTokenSpaceGuid.PcdCcEventlogAcpiTableLasa|0|UINT64|0x00010026
 
+  ## MU_CHANGE: Add a PCD to determine partition ID of TPM Service.
+  #
+  # Define the partition ID for TPM Service.
+  # Default PcdTpmServiceFfaPartitionId = 0x0, which should never be set to by
+  # the FF-A framework.
+  #
+  gEfiSecurityPkgTokenSpaceGuid.PcdTpmServiceFfaPartitionId|0|UINT16|0x0001002B

```

Comment:
Is this eventually going to be upstreamed? It looks like it might be part of broader FF-A changes?

Reply:
Yes, the plan is to upstream this change once the ARM patch set has settled. (along with some other changes on the FF-A topic)